cob_people_detection/ros/launch/head_detector.launch

```diff
@@ -1,15 +1,11 @@
 <?xml version="1.0"?>
 
 <launch>
-  <!-- head detection node (detects faces in depth image and publishes their positions) -->
-  <rosparam command="load" ns="/cob_people_detection/head_detector" file="$(find cob_people_detection)/ros/launch/head_detector_params.yaml"/>
-  <node name="head_detector" pkg="cob_people_detection" ns="/cob_people_detection/head_detector" type="head_detector_node" output="screen">
-    <!--remap from="pointcloud_rgb" to="/cam3d/depth/upright/points"/-->
-    <!--remap from="pointcloud_rgb" to="/cob_people_detection/image_flip/pointcloud_rgb_out"/-->  <!-- only activate on cob3 robots -->
```

Comment:
Please do not delete such valuable comments. If we ever want to use this again on a cob3 then it would take significantly more time if I have to figure out how image flip was integrated into the pipeline.